.github/workflows/e2e.yml

```diff
@@ -0,0 +1,63 @@
+# this workflow will run on every pr to make sure the project is following the guidelines
+
+# after labeler, run other actions with strict permissions
+name: CI
+
+on:
+  workflow_run:
+    workflows: [labeler]
+    types: [completed]
+    branches: ["*"]
+
+env:
+  TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+  TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
+
+jobs:
+  build-t3-app:
+    runs-on: ubuntu-latest
+    needs: labeler
```

Comment:
not sure if this will work if labeler is in a different file/workflow

Reply:
yea prob. the workflow only runs after the labeler anyways so